docs/md/source.md

```diff
@@ -95,6 +97,11 @@ The email addresses of those whom should be "readers" of all snapshots created
 by workflow-launcher in this workload. You can specify individual users and/or
 Terra/Firecloud groups.
 
+#### `pollingIntervalMinutes`
```

Comment:
Thanks for adding this!

You may also be able to remove this block higher up.

https://github.com/broadinstitute/wfl/pull/542/files#diff-90c5c39904e46ca35fcb11352d8ea5bc7dc34bd28dbebe4a1b37e2e170fb63afR25-R31